### ISSUE TITLE
Using segmentation masks (png files) like in torchvision mask rcnn ?

### ISSUE THREAD

**Issue by doursand** [facebookresearch/detectron2#742]
## ❓ Is there a way to use the same approach as for torchvision example (https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html)

In torchvision, one could use segmentation masks in png files to feed into the model for training where with Detectron2 , one need to use list[list[float]] for the segmentations. I had done that in another project and it worked fine

The problem I am facing is that the partner I am working with to annotate my images in polygon format (list[list[float]]) cannot apparently cope with the amount of samples I have in each of my images (around 900-1000!), their tool is crashing when they want to save the annotated polygons :-(

Therefore, they proposed me to generate instead segmentation masks and I could use that to train in torchvision, but I really (really) would like to stay with Detectron2 as it is far better from what I could experiment so far

Hence my question here , is there a way (out of the box) to use my images + their corresponding segmentation masks images to train my model ? Or should I develop something from scratch ?

Attached a picture of one of the segmentation mask image i was using with torchvision

![image](https://user-images.githubusercontent.com/37497557/72967903-aa664400-3dc2-11ea-841d-aded83042c01.png)

**Comment by ppwwyyxx** [comment]
Duplicate of #200
